### PR TITLE
topsql: update tidb topsql system variable doc

### PR DESCRIPTION
**Pull request by crazycs520**
Signed-off-by: crazycs520 <crazycs520@gmail.com>

### What is changed, added or deleted? (Required)

update tidb topsql system variable

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

https://github.com/pingcap/tidb/pull/31599

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1531,7 +1531,25 @@ set tidb_slow_log_threshold = 200;
...
 - 默认值：`ON`
-- 这个变量用控制是否开启 [Top SQL 特性](/dashboard/top-sql.md)。
+- 这个变量用于控制是否开启 [Top SQL 特性](/dashboard/top-sql.md)。
+
+### `tidb_top_sql_max_meta_count` <span class="version-mark">从 v6.0.0 版本开始引入</span>
+
+- 作用域：GLOBAL
+- 默认值：`5000`
+- 范围：`[1, 10000]`
+- 这个变量用于控制 [Top SQL](/dashboard/top-sql.md) 每分钟最多收集 SQL 种类的数量。
```
Comment (qiancai) [inline system-variables.md:1541]:
```suggestion
- 这个变量用于控制 [Top SQL](/dashboard/top-sql.md) 每分钟最多收集 SQL 语句类型的数量。
```

File: system-variables.md [hunk 2]
```diff
@@ -1531,7 +1531,25 @@ set tidb_slow_log_threshold = 200;
...
+
+### `tidb_top_sql_max_time_series_count` <span class="version-mark">从 v6.0.0 版本开始引入</span>
+
+- 作用域：GLOBAL
+- 默认值：`100`
+- 范围：`[1, 5000]`
+- 这个变量用于控制 [Top SQL](/dashboard/top-sql.md) 每分钟保留 Top-N 的 SQL 数据。
```
Comment (qiancai) [inline system-variables.md:1548]:
```suggestion
-  这个变量用于控制 [Top SQL](/dashboard/top-sql.md) 每分钟保留消耗负载最大的前多少条 SQL（即 Top N) 的数据。
```

File: system-variables.md [hunk 3]
```diff
@@ -1531,7 +1531,25 @@ set tidb_slow_log_threshold = 200;
...
+
+> **注意：**
+>
+> TiDB-Dashboard 中 TopSQL 界面目前总是只显示 Top-5 的数据，这与 `tidb_top_sql_max_time_series_count` 配置无关。
```
Comment (qiancai) [inline system-variables.md:1552]:
```suggestion
> TiDB Dashboard 中的 Top SQL 页面目前只显示消耗负载最多的 5 类 SQL 查询，这与 `tidb_top_sql_max_time_series_count` 的配置无关。
```

**Review by qiancai — COMMENTED** [review]
@crazycs520 PTAL 在 Top SQL 文档里是不是也可以介绍下这两个配置呢？

**Review by SunRunAway — APPROVED** [review]

File: system-variables.md [hunk 4]
```diff
@@ -1531,7 +1531,25 @@ set tidb_slow_log_threshold = 200;
...
+- 默认值：`5000`
+- 范围：`[1, 10000]`
+- 这个变量用于控制 [Top SQL](/dashboard/top-sql.md) 每分钟最多收集 SQL 语句类型的数量。
+
+### `tidb_top_sql_max_time_series_count` <span class="version-mark">从 v6.0.0 版本开始引入</span>
+
+- 作用域：GLOBAL
+- 默认值：`100`
+- 范围：`[1, 5000]`
+-  这个变量用于控制 [Top SQL](/dashboard/top-sql.md) 每分钟保留消耗负载最大的前多少条 SQL（即 Top N) 的数据。
```
Comment (SunRunAway) [inline system-variables.md:1548]:
```suggestion
- 这个变量用于控制 [Top SQL](/dashboard/top-sql.md) 每分钟保留消耗负载最大的前多少条 SQL（即 Top N) 的数据。
```

**Review by shichun-0415 — APPROVED** [review]

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**